src/app/templates/app/license_namespace_requests.html

```diff
@@ -0,0 +1,160 @@
+{% extends 'app/base.html' %}
+{% load static %}
+{% load staticfiles %}
+{% block body1 %}
+
+<link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/1.10.16/css/jquery.dataTables.css">
+
+<div id="messages" class="messages">
+</div>
+<p class ="lead"> {{ error }}</p>
+<div class="panel panel-default">
+<div class="panel-heading"> <p class="lead">License Namespace Requests List</p> </div>
+<div class="panel-body">
+    <table id="requests_table" class="display dataTable">
+      <thead>
+        <tr>
+          <th>Fullname</th>
+          <th>Short Identifier</th>
+          <th>License author name</th>
+          <th>Submission Date (UTC)</th>
+          <th>Archive</th>
+          <th>Promote</th>
+        </tr>
+      </thead>
+      <tbody>
+        {% if licenseNamespaceRequests %}
+        {% for elem in licenseNamespaceRequests %}
+          <tr class='clickableRow' id={{elem.id}}>
+            <td>{{elem.fullname}}</td>
+            <td>{{elem.shortIdentifier}}</td>
+            <td>{{elem.licenseAuthorName}}</td>
+            <td>{{elem.submissionDatetime}}</td>
+            <td>
+              {% csrf_token %}
+              <button id='archive_button{{elem.id}}' class='btn btn-success' width='25px' value='Archive'>Archive</button>
+            </td>
+            <td>
+              {% csrf_token %}
+              <button id='promote_button{{elem.id}}' class='btn btn-success' width='25px' {%if elem.promoted %} disabled {% endif %} {%if elem.promoted %} title="License request already created" {% endif %} value='Promote'>Promote to license list</button>
+            </td>
+          </tr>
+        {% endfor %}
+        {% endif %}
+      </tbody>
+    </table>
+
+{% include "app/modal.html" %}
+</div>
+{% endblock %}
+{% block script_block %}
+
+<script type="text/javascript">
+  $(document).ready(function () {
+      $("#licensenamespace").addClass('linkactive');
+      $("#licensenamespacerequests").addClass('linkactive');
+      });
+</script>
+
+<script type="text/javascript">
+  $(document).ready( function () {
+      $('#requests_table').DataTable({
+        'info': false,
+        'order': false
+      });
+  });
+</script>
+<script type="text/javascript" charset="utf8" src="https://cdn.datatables.net/1.10.16/js/jquery.dataTables.js"></script>
+
+<script type="text/javascript">
+
+  $('tbody tr').each(function (i,e) {
+    $(e).children('td:not(:last)').click(function () {
```

Comment:
Since you have added a `promote` button this should be updated, because now the archive button is not the last and when I click on the archive button it goes to the license namespace information page.

Reply:
I have corrected this.